spacy_llm/tasks/templates/sentiment.jinja

```diff
@@ -0,0 +1,22 @@
+Analyse whether the text surrounded by ''' is positive or negative. Respond with a float value between 0 and 1. 1 is strictly positive, 0 is strictly negative.
```

Comment:
The phrasing "strictly positive" and "strictly negative" sounds weird to me? As if talking about integers above and below zero. Would it be better to say something like "1 represents an extremely positive sentiment" or such?

Reply:
Updated in https://github.com/explosion/spacy-llm/pull/200/commits/b0e097722b9ae0ffe31d2232b6ff76156c1936a6.